src/parser/rogue/shared/constants.ts

```diff
@@ -1,38 +1,42 @@
 import SPELLS from 'common/SPELLS';
 import { SpellList } from 'common/SPELLS/Spell';
 
-export const SUBTLETY_ABILITY_COOLDOWNS: SpellList = {
-  ...SPELLS.SHADOWSTEP,
-  ...SPELLS.SHADOW_DANCE,
-  ...SPELLS.SYMBOLS_OF_DEATH,
-  ...SPELLS.SHADOW_BLADES,
-};
+const SHARED_ABILITY_COOLDOWNS: SpellList[] = [
+  SPELLS.CRIMSON_VIAL,
+  SPELLS.SPRINT,
+  SPELLS.KIDNEY_SHOT,
+  SPELLS.EVASION,
+  SPELLS.SHIV,
+  SPELLS.FEINT,
+  SPELLS.BLIND,
+  SPELLS.CLOAK_OF_SHADOWS,
+  SPELLS.TRICKS_OF_THE_TRADE,
+  SPELLS.SHROUD_OF_CONCEALMENT,
+];
 
-export const ASSASSINATION_ABILITY_COOLDOWNS: SpellList = {
-  ...SPELLS.SHADOWSTEP,
-  ...SPELLS.VENDETTA,
-};
+export const SUBTLETY_ABILITY_COOLDOWNS: SpellList[] = [
+  SPELLS.SHADOWSTEP,
+  SPELLS.SHADOW_DANCE,
+  SPELLS.SYMBOLS_OF_DEATH,
+  SPELLS.SHADOW_BLADES,
+  SHARED_ABILITY_COOLDOWNS,
+];
 
-export const OUTLAW_ABILITY_COOLDOWNS: SpellList = {
-  ...SPELLS.BETWEEN_THE_EYES,
-  ...SPELLS.BLADE_FLURRY,
-  ...SPELLS.ROLL_THE_BONES,
-  ...SPELLS.ADRENALINE_RUSH,
-  ...SPELLS.GRAPPLING_HOOK,
-};
+export const ASSASSINATION_ABILITY_COOLDOWNS: SpellList[] = [
+  SPELLS.SHADOWSTEP,
+  SPELLS.VENDETTA,
+  SHARED_ABILITY_COOLDOWNS,
+];
+
+export const OUTLAW_ABILITY_COOLDOWNS: SpellList[] = [
+  SPELLS.BETWEEN_THE_EYES,
+  SPELLS.BLADE_FLURRY,
+  SPELLS.ROLL_THE_BONES,
+  SPELLS.ADRENALINE_RUSH,
+  SPELLS.GRAPPLING_HOOK,
+  SHARED_ABILITY_COOLDOWNS,
+];
```

Comment:
Shouldn't the SHARED_ABILITY_COOLDOWNS be spread like ``...SHARED_ABILITY_COOLDOWNS`` ?

Reply:
I believe you are correct. Otherwise I'll have an embedded array.